### ISSUE TITLE
supervisor is still marked as non-support to Python 3 on https://python3wos.appspot.com

### ISSUE THREAD

**Issue by dalguji** [Supervisor/supervisor#510]
I see supervisor is now happily supporting Python 3 but it still appears marked as _non-support_ to Python 3 on https://python3wos.appspot.com and http://py3readiness.org

py3readiness.org mentions that it uses https://github.com/brettcannon/caniusepython3 to check availability of the package. We can see the online check for supervisor here: https://caniusepython3.com/project/supervisor

I am not sure why the online checker says supervisor has _1 blocker_ for python 3 port.

**Comment by flypenguin** [comment]
then please make a supervisor3 release, or something - the first SO question is almost 3 years old ...

**Comment by mnaberez** [comment]
Issues tagged [python 3](https://github.com/Supervisor/supervisor/labels/python%203) need to be resolved before any kind of release could be made.

**Comment by flypenguin** [comment]
that was not my suggestion :)

**Comment by mnaberez** [comment]
I read your comment.  I replied that the issues need to be resolved before any kind of release can be made (it could be called "supervisor3" like you suggested, or the existing package name, or whatever).  This software does not work reliably on Python 3 yet.  It doesn't matter what the package is called.  I'm not willing to release a version that I know is going to blow up and take down some production website.  When the Python 3 issues are actually resolved, I would be happy to make a release.

**Comment by nueverest** [comment]
Is there a projected release date for the python 3 support?

**Comment by johnrickman** [comment]
Bump

**Comment by loganbest** [comment]
bump

**Comment by yonatann** [comment]
bump

**Comment by ndmeiri** [comment]
Bump

**Comment by ahaerpfer** [comment]
bump

**Comment by viennadd** [comment]
bump

**Comment by jcyrss** [comment]
bump

**Comment by j-walker23** [comment]
Is there anywhere we can see a release schedule?

**Comment by infintesimal** [comment]
bump
I appreciate the hard efforts of the maintainers and would love to see them cut a tag of 4.0.0.

I have installed via pip from github master. I checked the github site and I don't see any open issues with the python3 label so would love to know what gates still need to be cleared before 4.0.0 can be tagged.

For what it's worth, I don't really think there is ever any defect free software. If some big production site blows up on py3 they will have the resources to revert or rollback. And any large scale site would be sure to test and stage this (they have their own risk management strategies and approaches). So if the vast majority of known issues are done, why not just put it out there with _caveat emptor_ and new reports can trickle in.
